DAQmxBaseApp/src/drvDaqMxBase.c

```diff
@@ -4164,6 +4198,10 @@ static void daqThread(void *param)
                 pInt32Interrupt = pNode->drvPvt;
                 reason = pInt32Interrupt->pasynUser->reason;
 
+                pInt32Interrupt->pasynUser->auxStatus = IOIntrStatusCode;
+                pInt32Interrupt->pasynUser->alarmStatus = IOIntrStatusCode;
+                pInt32Interrupt->pasynUser->alarmSeverity = IOIntrSeverityCode;
```

Comment:
We're not uniformly setting these everywhere that we set PV data. Do we also need to do it on lines:
* 3390
* 3830
* 4361
* 4539

Reply:
I've added writeIOIntrSeverity to the the interrupt callbacks which handle the data that has been collected, but I've left it out of the interrupts such as the STATE command as I'm not sure what might break if these go into invalid alarms.